index.html

```diff
@@ -231,5 +231,61 @@ <h2 class="aboutmehead">About me</h2>
 
 </article>
 </section>
+<footer class="footer">
+    <div class="containers">
+    <div  class="textheads" >
+    <h2 class="contactheads">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></div>
+    <form action="https://formspree.io/f/xrgdkqrd" method="post" class="myforms">
+        <ul class="textfields">
+            <li>
+                <hr class="liners">
+        <input type="text" placeholder="Full name" id="name" name="name" maxlength="30" required>
+        </li>
+        
+        <li>
+            <hr class="liners2">
+            <input type="email" placeholder="Email address" id="email" name="email-address" required >
+        </li>
+        <li>
+            <button type="submit" class="button button2">Get in touch</button>
+        </li>
+        <li>
+            <div class="textcontainer">
+            <textarea placeholder="Enter text here" id="textarea" name="comment" max-length="500" required></textarea>
```

Comment:
- [OPTIONAL] there is a typo here, I think it should be `maxLength` instead of `max-length` as you did in the name input 👍

- [OPTIONAL] I suggest removing the outline when focusing on the input
> To do that you can use
```
input:focus {
  outline: none;
}
```

> Great job using FormSpree and adding `required` attributes ✅💯